src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -166,10 +206,7 @@ private void buildDockerClient() {
    *     overlaps or conflicts with one passed into this method
    */
   private String startDockerContainerWithTerraInit(
-      String command,
-      String workingDir,
-      Map<String, String> envVars,
-      Map<String, File> bindMounts) {
+      String command, String workingDir, Map<String, String> envVars, Map<Path, Path> bindMounts) {
```

Comment:
it doesn't look like `workingDir` ever changes based on arguments. Instead of passing it through this and `startDockerContainer`, can we call `getWorkingDirOnContainer()` in `startDockerContainer`?

Reply:
Yes, done.